src/vl-multiselect.js

```diff
@@ -12,12 +12,17 @@ import {VlSelect} from '/node_modules/vl-ui-select/dist/vl-select.js';
  * @property {boolean} data-vl-error - Attribuut wordt gebruikt om aan te duiden dat het select element verplicht is of ongeldige tekst bevat.
  * @property {boolean} data-vl-success - Attribuut wordt gebruikt om aan te duiden dat het select element correct werd ingevuld.
  * @property {boolean} disabled - Attribuut wordt gebruikt om te voorkomen dat de gebruiker iets kan kiezen uit het select element.
- *
+ * @property {string} data-vl-placeholder - Attribuut bepaalt de placeholder in het zoek element.
```

Comment:
Waarom zit dit niet op niveau van vl-select?

Reply:
Nergens, omdat dat daar niet gebruikt werd.
Moet ik het naar daar verplaatsen?

Reply:
Ik denk dat deze placeholder niet specifiek voor vl-multiselect is, maar ook getoond wordt bij een vl-select dressed variant. In dat geval zou ik de logica inderdaad verplaatsen naar vl-select.